roles/mediawiki/files/special-log.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+# Make it so names of log types are more amenable to LFC users
+$logTitlesOverrideMessages = [
+  "picksomelogpage" => "Selection",
+  "activitylogpage" => "Misc. Activity (page views, login, logout, etc)",
+  "simplefavoriteslog" => "Favorites",
+  "log-name-create" => "Page Creation",
+  "movelogpage" => "Page Rename",
+  "uploadlogpage" => "File Upload",
+  "dellogpage" => "Page Deletion",
+  "newuserlogpage" => "User Creation",
+  "log-name-teamcomments" => "Comments",
+  "torquedataconnect-apiaccesslog" => "API Accesses",
+  "torquedataconnect-datachangeslog" => "Data Changes",
+  "rightslog" => "Changes to Access Control / Permissions"
+];
+
+$wgHooks['MessagesPreLoad'][] = function($title, &$message, $code) {
+  global $logTitlesOverrideMessages;
+  if(array_key_exists(strtolower($title), $logTitlesOverrideMessages)) {
+    $message = $logTitlesOverrideMessages[strtolower($title)];
+  }
+};
+
+
+# Remove log types that we don't want
```

Comment:
Yeah -- the comments in this file are really helpful.  This kind of commenting would be good to have too in `lfc-analysis-rule.php`.